Core/SaveState.cpp

```diff
@@ -46,6 +46,14 @@
 #include "HW/MemoryStick.h"
 #include "GPU/GPUState.h"
 
+#ifndef MOBILE_DEVICE
+#include "Core/AVIDump.h"
+#include "Core/HLE/__sceAudio.h"
+
+AVIDump video;
+WAVDump audio;
```

Comment:
Hmm, I think it'd be better to use `AVIDump::Stop();` and `WAVDump::Reset()` instead.  Since they are static methods, that can be used this way, without an instance.

It looks like it's being done the other way here, but it seems weird (at least it's a variable name less likely to be reused...):
https://github.com/hrydgard/ppsspp/blob/230f72a110b65fc75137ea92fa8689de1b264b71/UI/EmuScreen.cpp#L82

If using an instance like this, it really should have `static AVIDump video;` so that the video variable doesn't "leak" out.  Without static, it becomes a global variable for other modules and you can get conflicts if someone else uses `video`.

-[Unknown]